SECURITY.md

```diff
@@ -14,17 +14,17 @@ Instead, please report them to the Microsoft Security Response Center (MSRC) at
 
 If you prefer to submit without logging in, send email to [secure@microsoft.com](mailto:secure@microsoft.com).  If possible, encrypt your message with our PGP key; please download it from the [Microsoft Security Response Center PGP Key page](https://www.microsoft.com/en-us/msrc/pgp-key-msrc).
```

Comment:
```suggestion
If you prefer to submit without logging in, send email to [secure@microsoft.com](mailto:secure@microsoft.com).  If possible, encrypt your message with our PGP key; please download it from the [Microsoft Security Response Center PGP Key page](https://www.microsoft.com/msrc/pgp-key-msrc).
```

Reply:
you missed this one, it's updated in the security.md we take from

Reply:
Actually, I didn't, but I [was told to revert](https://github.com/microsoft/PowerToys/pull/24376#issuecomment-1453845520) it 🙃
Question, though: How do we "take" that? This is a [quote ](https://github.com/microsoft/.github/pull/62#issuecomment-1256431827)from jeffwilcox:

> It will not be updated in repos, and we will not make an explicit push to update it, it's very noisy. However, net new repos will get it.

Reply:
> How do we "take" that?

I don't know, tbh. I just learned that as well :)

Reply:
Maybe just as simple as copy the entire file over.. But that will be shown as same as you did it if you don't explicitly note in description that you copied the file :) @Aaron-Junker any idea?

Reply:
I think we should keep the one we have right now as it is and when OSPO updates theirs we can copy it to our repo.

Reply:
@Jay-o-Way Aaron is right, let's revert security.md. Those are autoformatting changes anyway, and keep this only loc related.

We can even probably automate update of security.md as a commit hook or CI step.